dockerfiles/backfill_storage/Dockerfile

```diff
@@ -17,8 +17,9 @@ RUN go build
 # build mcd with local vdb
 WORKDIR /go/src/github.com/makerdao/vdb-mcd-transformers
 RUN go mod edit -replace=github.com/makerdao/vulcanizedb=/go/src/github.com/makerdao/vulcanizedb/
-RUN make plugin PACKAGE=github.com/makerdao/vdb-mcd-transformers
-
+RUN make plugin PACKAGE=github.com/makerdao/vdb-mcd-transformers \
+    TARGET_LOCATION=$GOPATH/src/github.com/makerdao/vdb-mcd-transformers/plugins/execute/transformerExporter.go \
```

Comment:
Should this `TARGET_LOCATION` be pointing at `.../plugins/backfill/...` instead of `.../plugins/execute/...`?

Reply:
Not at the moment, but this is definitely a bit confusing. `plugins/backfill` is used by `backfill_events` and has no storage transformers. I wonder if it would make sense to have new exporters for both `plugins/backfill_events` and a `plugins/backfill_storage` - since we probably don't need all of the transformers from `execute` when running `backfill_storage`. Open to other ideas

Reply:
Splitting backfill_events and backfill_storage makes sense to me!